cli/formatter/table.go

```diff
@@ -283,9 +283,11 @@ func renderEqualMaps(maps []unorderedMap[any], transpose bool, opts Opts) (strin
 	t := newTableWriter(opts)
 
 	var commonKeys []any
-	maps[0].forEach(func(mapKey, _ any) {
-		commonKeys = append(commonKeys, mapKey)
-	})
+	if len(maps) > 0 {
+		maps[0].forEach(func(mapKey, _ any) {
+			commonKeys = append(commonKeys, mapKey)
+		})
+	}
```

Comment:
It will be obvious:
```suggestion
	if item, ok := maps[0]; ok {
		item.forEach(func(mapKey, _ any) {
			commonKeys = append(commonKeys, mapKey)
		})
	}
```

Reply:
`maps` is a slice